apps/style/JsDebuggerUi.scss

```diff
@@ -44,13 +44,17 @@
   opacity: 1;
 }
 
+// When opening the debug console with the ellipsis, the codeTextbox sometimes
+// renders over the debug console. Setting the z-index to 2 is a work-around for
+// this issue. See JsDebugger.jsx.
```

Comment:
https://codedotorg.atlassian.net/browse/STAR-595

Reply:
Thanks for capturing this issue. Even though it is harder, I think we should do the 'hard' solution that you propose in that ticket.

Reply:
I agree. The "easy" option might just be a stop-gap while we work on the "hard" option.